src/config/algolia.js

```diff
@@ -0,0 +1,5 @@
+import algoliasearch from 'algoliasearch';
+
+const algolia = algoliasearch(process.env.ALGOLIA_APP_ID, process.env.ALGOLIA_API_KEY);
```

Comment:
I can't test your branch since  I don't have an id nor a key. Could you pm them to us? :smile:

Reply:
Maybe you could add ALGOLIIA_APP_ID + KEY to `.env.dist` without the actual keys, just so everyone knows about the required addition? :smiley:

Reply:
And perhaps just write a few lines about how algolia is part of the project in `README.md` - so that potential new developers know whats going on and what the `/algolia` route does? :+1: